volume/btrfs/btrfs.go

```diff
@@ -34,6 +35,13 @@ const (
 	DriverName = "btrfs"
 )
 
+var (
+	ErrBtrfsCommand      = errors.New("error running btrfs command")
```

Comment:
this should perhaps be a type where you can include the command being called along with the output of the command that implement error